sdk/python/kfp/compiler/compiler.py

```diff
@@ -681,6 +681,12 @@ def _create_pipeline_workflow(self, args, pipeline, op_transformers=None, pipeli
     if pipeline_conf.ttl_seconds_after_finished >= 0:
       workflow['spec']['ttlSecondsAfterFinished'] = pipeline_conf.ttl_seconds_after_finished
 
+    if pipeline_conf.min_available != None:
+      poddisruptionbudget = {"minavailable": pipeline_conf.min_available}
```

Comment:
readability nit:
```suggestion
      pod_disruption_budget = {'min_available': pipeline_conf.min_available}
```
Same below. We should try to keep the code style consistent. Thanks!

Reply:
Sound good! Will make manual adjust instead of the suggestion function to avoid issues with the CLA bot which seems to have some issues: https://github.com/kubeflow/pipelines/pull/4199#issuecomment-658720833

Reply:
`minavailable` should probably not have underscore. But it's better to use camelCase: `minAvailable`. I did not know Kubernetes was case insensitive.

Reply:
Yeah I  realised that and changed to ```minAvailable```, ``` podDisruptionBudget```, as the keys in the dicts that are passed to workflow specs.

Reply:
Oh thanks for the catch.